core/controllers/library_test.py

```diff
@@ -194,14 +197,138 @@ def test_library_handler_for_created_explorations(self):
         }, response_dict['activity_list'][0])
 
 
+class LibraryIndexHandlerTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(LibraryIndexHandlerTests, self).setUp()
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+        self.viewer_id = self.get_user_id_from_email(self.VIEWER_EMAIL)
+
+    def test_library_index_handler_for_user_preferred_language(self):
+        """Test the access to user's preferences."""
```

Comment:
`Test ability to change the language in user's preferences.`

Reply:
Hi, thanks for reviewing! 

In my opinion, "Test ability to change the language in user's preferences." is not the goal of this test, although it does so. In this test, I want to test if the handler gets the preference correctly when the user logs in. The reason I change the language is that the language preference would be "en" whether the user logs in or not.

Reply:
Then `Test whether the handler returns the correct language preference` would be a better description. Also, do put a comment as mentioned above in the test on why the language preference is changed.

Reply:
[Fixed in this commit](https://github.com/oppia/oppia/pull/6760/commits/95f7a4b0a3268af9334af5256c10ad6a8bf17ae7)

Thank you!